test/freecoin/test_helpers/store.clj

```diff
@@ -1,6 +1,8 @@
 (ns freecoin.test-helpers.store
-  (:import [freecoin.db.mongo MemoryStore]
-           [freecoin.blockchain InMemoryBlockchain]))
+  (:require [freecoin-lib.db.mongo :as db]
+            [freecoin-lib.core :as bc])
+  (:import [freecoin_lib.db.mongo MemoryStore]
+           [freecoin_lib.core InMemoryBlockchain]))
 
```

Comment:
The TestStore is not used anywhere as far as I can tell, can we remove the file?